hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -634,6 +632,23 @@ private TokenPauseStatus tokenPauseStatusOf(final boolean flag) {
 		return relationships;
 	}
 
+	public static List<TokenID> getAssociatedTokens(
```

Comment:
Won't it be much more efficient to construct the `TokenRelationship`s in the `while()` below, instead of first constructing a list of `TokenID`s; and then repeating a call to `tokenAssociations.get()` above for each `TokenID`?

Reply:
yes, I was thinking in the lines of reusing this logic in places where we need size as well. Certainly better to save the count in the merkleAccountState and just build tokenRelationships here